### PR TITLE
remove read_quorum

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

File: tests/storage/test_raftkv.rs [hunk 1]
```diff
@@ -85,7 +65,7 @@ fn test_read_leader_in_lease(read_quorum: bool) {
     cluster.add_send_filter(IsolationFilterFactory::new(leader.get_store_id()));
 
     // leader still in lease, check if can read on leader
-    assert_eq!(can_read(&ctx, storage.as_ref(), k2, v2), !read_quorum);
+    assert_eq!(can_read(&ctx, storage.as_ref(), k2, v2), true);
```
Comment (siddontang) [inline tests/storage/test_raftkv.rs]: is it ok here?  we may not read v2 here.

Comment (zhangjinpeng1987) [inline tests/storage/test_raftkv.rs]: It's ok here.

**Comment by siddontang** [comment]
CI failed

**Comment by disksing** [comment]
LGTM.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**